lib/tasks/projects.rake

```diff
@@ -15,14 +15,15 @@ namespace :projects do
     users = tenk.users.list(per_page: 100).data
 
     users.each do |user|
-      puts user.display_name
-      team_member = TeamMember.find_or_create_by!(
+      puts user
+      team_member = TeamMember.find_or_initialize_by(tenk_id: user.id)
```

Comment:
Does this line solve the problem of running the rake task when you haven't drop the DB and getting an error saying the `tenk_id` already exists?

Reply:
Yes - previously it was looking for a `TeamMember` which exactly matched every attribute given. If one didn't exist (eg if the name had changed) then it would try to create a new one, the `tenk_id` would collide and it would cause an exception.

Here it's only matching on the one thing - `tenk_id` - and then overwriting everything else.